wp/makefile

```diff
@@ -0,0 +1,5 @@
+all:
+	xelatex wp.tex
```

Comment:
How did you test this command? I tried to run `make all` on Ubuntu, and it entered interactive mode with a bunch of errors.

Reply:
Maybe [this](https://www.overleaf.com/learn/latex/chinese#CJKutf8_with_pdfLaTeX) could help?
> this is usually only convenient for documents in English with bits of Chinese text

Reply:
Yes, thank you!